platform/persistence/elastic/bundle.json

```diff
@@ -23,7 +23,7 @@
             },
             {
                 "key": "ELASTIC_ROOT",
-                "value": "http://localhost:9200",
+                "value": "/elastic",
                 "priority": "fallback"
```

Comment:
This seems like an unrelated change - is there a reason this was included, or did it just tag along with another commit?